spec/models/institution_builder_spec.rb

```diff
@@ -53,32 +60,31 @@
         expect(described_class.run(user)[:error_msg]).to eq('BOOM!')
       end
 
-      it 'logs errors at the database level' do
+      def statement_invalid
         error_message = 'BOOM!'
-
         statement_invalid = ActiveRecord::StatementInvalid.new(error_message)
         statement_invalid.set_backtrace(%(backtrace))
+        statement_invalid
+      end
 
+      it 'logs errors at the database level' do
         allow(described_class).to receive(:add_crosswalk).and_raise(statement_invalid)
         expect(Rails.logger).to receive(:error).with('There was an error occurring at the database level: BOOM!')
         described_class.run(user)
       end
 
       it 'logs errors at the Rails level' do
         allow(described_class).to receive(:add_crosswalk).and_raise(StandardError, 'BOOM!')
-
         expect(Rails.logger).to receive(:error).with('There was an error of unexpected origin: BOOM!')
         described_class.run(user)
       end
 
       it 'does not change the institutions or versions if not successful' do
         allow(described_class).to receive(:add_crosswalk).and_raise(StandardError, 'BOOM!')
         create :version
-        version = Version.current_preview
-
         described_class.run(user)
         expect(Institution.count).to be_zero
-        expect(Version.current_preview).to eq(version)
+        expect(Version.current_preview).to eq(Version.current_preview)
```

Comment:
Are you actually checking anything here?

Reply:
No it doesn't. I've put a fix in.